lib/reporters/cli/index.js

```diff
@@ -200,31 +230,7 @@ PostmanCLIReporter = function (emitter, reporterOptions, options) {
         timings = _.last(_.get(o, 'history.execution.data'));
         timings = timings && timings.timings;
 
-        // print timing info of the request
```

Comment:
We cannot remove this part, because this is essential for `newman run`, we will have to put conditionals using the flag for `singleRequest`

Reply:
@raghavbhatia336 Just a thought, can we use this as default for showing timing phases instead of our own table? This looks neater to me.